content/blog/FOSDEM-2020/index.md

```diff
@@ -0,0 +1,23 @@
+---
+date: "2020-01-01T09:00:00+00:00"
+draft: false
+weight: 180
+title: "Syndesis at FOSDEM 2020"
+menu:
+  topnav:
+    parent: blog
+---
+
+<p class="lead">Join us at <a href="https://fosdem.org/2020/" rel="nofollow" target="_blank">FOSDEM 2020</a>, we'll be at the <a href="https://fosdem.org/2020/stands/" rel="nofollow" target="_blank">Integration Community Stand</a> both days of the conference</p>
+
+{{< figure src="fosdem2020-box.png" alt="FOSDEM 2020" link="https://fosdem.org/2020/" rel="nofollow" target="_blank" class="aside" >}}
+
+FOSDEM is two-day free, no registration, just show up, event in Brussels, Belgium, happening each year on the first weekend of February. It is the biggest community event around free/open source software and a great place to meet in person and exchange ideas.
+
+This year FOSDEM is on the 1st and 2nd of February at the [Université libre de Bruxelles Campus du Solbosch](https://fosdem.org/2020/practical/transportation/).
+
+The Syndesis project will be represented at the [Integration Community Stand](https://fosdem.org/2020/stands/) which we share with the [Apache Camel](https://camel.apache.org/blog/FOSDEM-2020/) community.
```

Comment:
Ideally all links would have `rel='nofollow'` and `target='blank'` but that's just nitpicking 😝

Reply:
Oh that should have been taken care of by this bit: https://github.com/syndesisio/syndesis.io/pull/141/files#diff-0f1be8eec416e71c28b41efb0544df26R14-R15